lib/markdownlint-provider.coffee

```diff
@@ -15,10 +18,19 @@ module.exports =
 
       bufferText = textEditor.getText()
 
+      filePath = textEditor.getPath()
+
       options = {
         "strings": { "string": bufferText }
       }
 
+      configPath = find filePath, '.markdownlintrc'
+
+      if configPath
+        configText = fs.readFileSync(configPath, 'utf8')
+        options['config'] = rc.parse(configText)
```

Comment:
I don't like to add unnecessary dependencies, isn't `.markdownlintrc` just a JSON file?